pvlib/tests/test_modelchain.py

```diff
@@ -693,6 +705,23 @@ def test_run_model_with_weather_fuentes_temp(sapm_dc_snl_ac_system, location,
     assert not mc.results.ac.empty
 
 
+def test_run_model_with_weather_noct_sam_temp(sapm_dc_snl_ac_system, location,
+                                              weather, mocker):
+    weather['wind_speed'] = 5
+    weather['temp_air'] = 10
+    sapm_dc_snl_ac_system.temperature_model_parameters = {
```

Comment:
https://github.com/pvlib/pvlib-python/pull/1196 will need to update for this one too